internal/jsre/deps/web3.js

```diff
@@ -2307,7 +2307,7 @@ var toChecksumAddress = function (address) {
 };
 
 /**
- * Transforms given string to valid 20 bytes-length addres with 0x prefix
+ * Transforms given string to valid 20 bytes-length address with 0x prefix
```

Comment:
this change should be sumbitted to web3 js not here this file is updated from there.
Skip this one